malaria24/ona/models.py

```diff
@@ -654,9 +655,20 @@ class ReportedCase(models.Model):
     ehps = models.ManyToManyField('Actor', blank=True)
     form = models.ForeignKey('OnaForm', null=True, blank=True)
 
+    def normalize_msisdn(self, mobile_number):
+        valid_prefix = '[+27]'
```

Comment:
If you define this variable without the `[]`, you can use it on line 663 too instead of `'+27' + ...`

Reply:
I tried without the `[]` but because it needs to go into regex, it somehow needs the square brackets. I could however remove the variable and add the [+27] into the regex string

Reply:
You will probably have to add the `[]` into the regex string on line 660 then.

`if re.match('^[' + valid_prefix + ']*([0-9]{9})$', mobile_number):`

Reply:
I changed the regex string to `'^[+27]*([0-9]{9})$'`, removed the variable, and it works. Hope that makes it okay